docs/collections/graphs/quickstart.md

```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
 - INBOUND: `_from` ← `_to`
 - ANY: `_from` ↔ `_to`.
 
-## Example
+## Prerequisites
 
-For this example, assume the following credentials:
+- Access to a [Macrometa account](https://auth.paas.macrometa.io/) with sufficient permissions to create streams.
+- Have your API Key ready. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).
+- Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
 
-- Tenant name is `nemo@nautilus.com`.
-- User password is `xxxxxx`.
+### Step 1. Connect to GDN
```

Comment:
```suggestion
### Connect to GDN
```

I would either remove "Step 1" or add "Step 2," "Step 3," etc. to the rest of the headers.

Reply:
Check the other quickstarts and getting started topics. I think this is consistent with them.

Also, we need to rewrite the whole thing anyway.

Reply:
Dallas, don't worry about the step numbers for now. James and I will deal with it when we test and rewrite this quickstart.

Reply:
Yeah, after posting my last comment I saw we have other Quickstarts similarly formatted and never edited my comment.  We can make it consistent when we rewrite.